src/textInput.test.jsx

```diff
@@ -11,34 +10,32 @@ describe('TextInput', function() {
 		MAX_LEN = '20',
 		ERROR_TEXT = 'Too wimpy.';
 
-	let textInput,
-		textInputEl,
+	let textInputComponent,
 		inputEl;
 
 	beforeEach(() => {
 		const formAttrs = {
 			id: NAME_ATTR,
 			maxLength: MAX_LEN,
 			error: ERROR_TEXT,
-			required: 'required'
+			required: true
 		};
-		textInput = TestUtils.renderIntoDocument(<TextInput
+		textInputComponent = TestUtils.renderIntoDocument(<TextInput
 			name={NAME_ATTR}
 			label={LABEL_TEXT}
 			value={VALUE}
 			{...formAttrs} />);
 
-		textInputEl = ReactDOM.findDOMNode(textInput);
-		inputEl = textInputEl.querySelector('input');
+		inputEl = TestUtils.findRenderedDOMComponentWithTag(textInputComponent, 'input');
 	});
 
 	afterEach(() => {
-		textInputEl = null;
+		textInputComponent = null;
 		inputEl = null;
 	});
 
 	it('exists', () => {
-		expect(textInputEl).not.toBeNull();
+		expect(inputEl).not.toBeNull();
```

Comment:
Unfortunately this test will not capture the error property, you will want to do something like this instead (event for tag) as findRenderDOM always expects to find at least 1 element or throws an error. Where as https://facebook.github.io/react/docs/test-utils.html#scryrendereddomcomponentswithtag will return an empty array and you can instead test the array length. Both are good options for this.